MULTIPLATFORM.md

```diff
@@ -0,0 +1,244 @@
+# Kotlin/Native in multiplatform projects
+
+*TODO: Some introduction*
+
+## Creating multiplatform Android/iOS application with Kotlin
+
+*TODO: High-level application structure: what code is common (data, logic) and what code is platform (io, ui, network).
+What languages are used in what parts of the project.*
+
+*TODO: link to the calculator project*
+
+*TODO: link to the multiplatform description*
+
+### 1. Preparing a workspace
+
+Our multiplatform application will include three parts:
+
+ * An **Android application** represented by a separate Android Studio project.
+ * An **iOS application** represented by a separate XCode project.
+ * A **multiplatform library** represented by a separate Gradle-build and containing a business logic of the application.
+   This library can contain both platform-dependent and platform-independent code and is compiled into a `jar`-library
+   for Android and in a framework for iOS.
+
+In its turn, the multiplatform library will include three subprojects:
+
+ * `common` - contains a common logic for both applications;
+ * `ios` - contains an iOS-specific code;
+ * `android` - contains an Android-specific code.
+
+Represent this structure as a directory tree. Assume that our multiplatform library is intended to generate different
+greetings on different platform. Create the following directory structure:
+
+    application/
+    ├── androidApp/
+    ├── iosApp/
+    └── greeting/
+        ├── common/
+        ├── android/
+        └── ios/
+
+Now we have a basic structure of the multiplatform application and can proceed to implementing of the multiplatform library.
+
+### 2. Multiplatform library
+
+*Note: It's highly recommended to use [Gradle wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html)
+to run the build. To create the wrapper, just execute `gradle wrapper` in the `greeting` directory. After that you can
+use `./gradlew` to run the build instead of using your local Gradle installation.*
+
+The multiplatform library is build using Gradle so the first thing we need to do is to specify the structure of the
+project. To do this, create `settings.gradle` and declare in it all the subprojects:
```

Comment:
Better to say the directory where it has to be fined ( in the `greeting/` directory)
(Like you have done with the `build.gradle` file below 👇 )

Reply:
Why root gradle project is at the library directory? Why not in the application root? What if I'd want to add another mp-library later?

Reply:
> Why root gradle project is at the library directory? Why not in the application root?

The initial idea was to have absolutely independent projects for all parts of the application (Android app, iOS app and the library). Having Android and iOS projects in subdirectories of a root Gradle build makes such a separation less clear and may confuse people.

But you comment about several mp-libraries makes sense so I think we can move the root Gradle project into the root directory.

Reply:
> is built

Fixed